Modules/Tracker/QuestieTrackerUtils.lua

```diff
@@ -225,8 +225,8 @@ function QuestieTracker.utils:FlashFinisher(Quest) -- really terrible animation
     end)
 end
 
--- function QuestieTracker.utils:FlashObjectiveByTexture(Objective) -- really terrible animation code, sorry guys
---     if Objective.AlreadySpawned then
+-- function QuestieTracker.utils:FlashObjectiveByTexture(objective) -- really terrible animation code, sorry guys
+--     if objective.AlreadySpawned then
```

Comment:
Remove this long comment if that stuff is not used

Reply:
Yeah, I saw that but I only added a couple items to the bottom of this file. It was like that when I made the change. We should check with the rest of the Dev team to make sure it's not going to be added in the future to be part of a new feature?

Reply:
Yeah that is fine with me.